contracts/ERC20Safe.sol

```diff
@@ -291,4 +316,24 @@ contract ERC20Safe is BridgeRole, Pausable {
         Batch memory batch = batches[batchesCount - 1];
         return _isBatchProgessOver(batch.depositsCount, batch.blockNumber) || batch.depositsCount >= batchSize;
     }
+
+    function _internalMint(address token, address to, uint256 amount) internal returns (bool) {
+        IMintableBurnableERC20 mintableToken = IMintableBurnableERC20(token);
+
+        if (!mintableToken.hasRole(mintableToken.MINTER_ROLE(), address(this))) {
```

Comment:
I don't think we need to check this here - it's the token's job to check the minter role and perform appropriate logic - also, if this would revert it would crash the executeTransfers loop from the bridge contract.
  `mint` and `burn` are enough for the safe to perform it's logic

Reply:
if we dont have the mint role, the whole bridge is blocked until the owner will give us the appropiate role, thats why i propose to reject the transfer for which we dont have the role